### PR TITLE
copy all the text files if anafora dir is empty

### PR DESCRIPTION
**Pull request by EgoLaparra**
A tentative proposal. The idea is to extract all the text files from `text_dir` if `anafora_dir` is empty.

### CODE REVIEW

File: anafora/copy_text.py [hunk 1]
```diff
@@ -10,17 +10,22 @@
 
 def copy_timeml_text(text_dir, anafora_dir, xml_name_regex, write_dct):
     text_name_to_path = {}
+    text_name_to_folder = {}
     for dir_path, _, file_names in os.walk(text_dir):
         for file_name in file_names:
             if file_name.endswith(".tml"):
                 file_path = os.path.join(dir_path, file_name)
                 text_name_to_path[file_name[:-4]] = file_path
+                file_folder = dir_path.replace(text_dir, '')
```
Comment (bethard) [inline anafora/copy_text.py]: Does this work if `text_dir` does not end in a `/`? Maybe use `os.path.relpath` instead? (Same comment for the similar lines below).

**Review by bethard — COMMENTED** [review]
This approach looks good to me. See my one comment about using `os.path.relpath`.

File: anafora/copy_text.py (hunk 1 above)
Comment (EgoLaparra) [reply anafora/copy_text.py]: Changed!

**Review by bethard — APPROVED** [review]
Looks good.

**Pull request merged**